transactions/Eternal/pay.cdc

```diff
@@ -8,7 +8,7 @@ transaction(amount: UFix64) {
         self.sentVault <- vaultRef.withdraw(amount: amount)
     }
     execute {
-        let recipient = getAccount(0x1f56a1e665826a52)
+        let recipient = getAccount(0xETERNAL_PAYMENT_ADDRESS)
```

Comment:
can someone verify this will be replaced with the value in `config.json` before merging? thanks

Reply:
yes this will